### ISSUE TITLE
Error accessing isnull() and notnull() on Dataset

### ISSUE THREAD

**Issue by duncanwp** [pydata/xarray#1715]
#### Code Sample, a copy-pastable example if possible

```python
# Create basic Dataset
import numpy as np
import pandas as pd
import xarray as xr

temp = 15 + 8 * np.random.randn(2, 2, 3)

precip = 10 * np.random.rand(2, 2, 3)
lon = [[-99.83, -99.32], [-99.79, -99.23]]
lat = [[42.25, 42.21], [42.63, 42.59]]

# for real use cases, its good practice to supply array attributes such as
# units, but we won't bother here for the sake of brevity
ds = xr.Dataset({'temperature': (['x', 'y', 'time'],  temp),
                 'precipitation': (['x', 'y', 'time'], precip)},
                coords={'lon': (['x', 'y'], lon),
                        'lat': (['x', 'y'], lat),
                        'time': pd.date_range('2014-09-06', periods=3),
                        'reference_time': pd.Timestamp('2014-09-05')})

# Both of these throw attribute errors
ds.notnull()
# ds.isnull()
```
#### Problem description

The above methods throw `AttributeError: 'Variable' object has no attribute '_constructor'` errors when called on Datasets. It's not clear if this operation is currently supported, but if it's not then it probably shouldn't be exposed through the API.

#### Expected Output

Dataset of boolean DataArrays indicating null-ness.  

#### Output of ``xr.show_versions()``

<details>
Show_versions() doesn't seem to exist in 0.9.6, but presumably this covers it:

Numpy version: 1.13.1
Pandas version: 0.21.0
XArray version: 0.9.6
</details>

**Comment by jhamman** [comment]
@duncanwp - can you try updating to the v0.10 release candidate. I think should fix this issue.

```
pip install --pre --upgrade --upgrade-strategy=only-if-needed xarray
```

**Comment by jhamman** [comment]
This is likely a duplicate of #1663, and was fixed by #1549.

**Comment by duncanwp** [comment]
Yes, you're right - it works now in 0.10.0rc1. Apologies for the noise!
